### DISCUSSION TITLE
Entering PreAlpha!

### DISCUSSION THREAD

**Discussion by GuySandler** [GuySandler/LockerCam#5] [Announcements]
Thanks to @Keshava11 for the code!

**Comment by Keshava11** [comment]
>  Thanks to @Keshava11 for the code!

I didn't get that.
Which code are we talking about ?

**Reply by GuySandler** [reply]
The one called FileSender0point1Public.py, you made the script about 5 years ago for sending files to Dropbox using Python, and I edited it to connect to my app.

**Reply by Keshava11** [reply]
Oh okay that gist. 
It had been a while.
Glad that you found it useful.
All the best 👍
